.github/workflows/west-commands.yml

```diff
@@ -1,9 +1,11 @@
 name: NCS west commands
 
 on:
+  workflow_dispatch:
   pull_request:
     branches: [main]
     paths:
+      - scripts/west_commands/sbom/**
```

Comment:
we should add: 
- 'scripts/scripts/requirements-west-ncs-sbom.txt'
- 'scripts/scripts/requirements-extra.txt'
- 'applications/asset_tracker_v2/**/*.c'

Reply:
Added requirements but I think there is not much use for triggering in case these changes: 'applications/asset_tracker_v2/**/*.c